docs/contract-test-collection.md

```diff
@@ -120,19 +193,49 @@ storage (pair
           (nat %totalSupply));
 ```
 
+</TabItem>
+  <TabItem value="taquito">
+
+```js
+const allowances = new MichelsonMap();
+    const ledger = new MichelsonMap();
+    ledger.set('tz1btkXVkVFWLgXa66sbRJa8eeUSwvQFX4kP', { allowances, balance: '100' });
+
+    const opknownBigMapContract = await tezos.contract.originate({
+      code: knownBigMapContract,
+      storage: {
+        ledger,
+        owner: 'tz1gvF4cD2dDtqitL3ZTraggSR1Mju2BKFEM',
+        paused: true,
```

Comment:
Based on the michelson type, there is no `paused` in the storage of the contract.

Reply:
I removed the line. thanks